app/serializers/api/v3/etablissement_serializer.rb

```diff
@@ -4,4 +4,12 @@ class API::V3::EtablissementSerializer < ApplicationSerializer
   attributes *all_fields
 
   belongs_to :unite_legale
+
+  def etablissement_siege
+    if ['true', 't'].include?(object.etablissement_siege)
+      true
+    elsif ['false', 'f'].include?(object.etablissement_siege)
+      false
+    end
```

Comment:
```suggestion
    ['true', 't'].include?(object.etablissement_siege)
```

Plutôt éviter les `if true return true else return false`

Reply:
Il existe un cas non géré dans ton fix : `nil`, du coup compliqué de faire "propre" ici..

Reply:
@skelz0r c'est pour ça que je voulais ajouter un return `nil` explicite !

Reply:
Ça me va pour ce cas ça s'entend